local/src/core/YamlParser.cc

```diff
@@ -514,10 +528,19 @@ YamlParser::parse_fields_and_rules(
             values.emplace_back(localized_value);
             fields.add_field(name, localized_value);
           }
-          if (auto const rule_type_node{ValueNode[YAML_RULE_TYPE_MAP_KEY]}; rule_type_node) {
+          auto const rule_case_node{ValueNode[YAML_RULE_CASE_MAP_KEY]};
+          bool is_nocase = false;
+          if (rule_case_node && rule_case_node.IsScalar()) {
+            TextView case_str = Localizer::localize(rule_case_node.Scalar());
+            if (case_str == VERIFICATION_DIRECTIVE_IGNORE) {
+              is_nocase = true;
+            }
+          }
+          auto const rule_type_node = ValueNode[YAML_RULE_TYPE_MAP_KEY];
+          if (rule_type_node) {
             TextView rule_type{rule_type_node.Scalar()};
             std::shared_ptr<RuleCheck> tester =
-                RuleCheck::make_rule_check(name, std::move(values), rule_type);
+                RuleCheck::make_rule_check(name, std::move(values), rule_type, false, is_nocase);
```

Comment:
I avoid `false` and `true` as parameters to functions. They're essentially magic values that make it harder for the person reading the function call to understand what is happening (in the future I'll say, "What does `false` here mean again?"). In the header file, can you please create `constexpr bool IS_INVERTED = true` and `constexpr bool IS_NOCASE = true` variables that can be used here?

Reply:
Reworked to eliminate call with false.